src/tools/tidy/src/primitive_docs.rs

```diff
@@ -0,0 +1,17 @@
+//! Tidy check to make sure `library/{std,core}/src/primitive_docs.rs` are the same file.  These are
+//! different files so that relative links work properly without having to have `CARGO_PKG_NAME`
+//! set, but conceptually they should always be the same.
```

Comment:
Something like this comment is what I was thinking for putting at the top of `primitive_docs.rs`.